denonavr/const.py

```diff
@@ -262,6 +264,12 @@
 ZONE2 = "Zone2"
 ZONE3 = "Zone3"
 VALID_ZONES = [MAIN_ZONE, ZONE2, ZONE3]
+ZONE_SOURCES = ["CD", "PHONO", "TUNER", "DVD", "BD", "TV", 
+    "SAT/CBL","MPLAY","GAME","HDRADIO","NET","PANDORA",
+    "SIRIUSXM","LASTFM","FLICKR","IRADIO","SERVER","FAVORITES",
+    "AUX1","AUX2","AUX3","AUX4","AUX5","AUX6","AUX7","BT","USB/IPOD",
+    "USB DIRECT", "IPOD DIRECT"
+]
```

Comment:
Please call this constant `TELNET_SOURCES` and put it right behind `CHANGE_INPUT_MAPPING` constant.